package.json

```diff
@@ -105,21 +100,18 @@
     "ember-router-generator": "^1.0.0",
     "escape-string-regexp": "^1.0.3",
     "exists-sync": "0.0.3",
-    "exit": "^0.1.2",
     "express": "^4.12.3",
     "findup": "0.1.5",
     "findup-sync": "^0.2.1",
     "fs-extra": "0.22.1",
     "fs-monitor-stack": "^1.0.2",
     "git-repo-info": "^1.0.4",
-    "github": "^0.2.3",
```

Comment:
This is used here: https://github.com/ember-cli/ember-cli/blob/master/bin/changelog#L17